### ISSUE TITLE
Don't use mutable function parameters

### ISSUE THREAD

**Issue by raphael0202** [tducret/trainline-python#8]
In `Passenger.__init__`, a mutable parameter is used for `cards`, which leads to the following bug:
```
import trainline

passenger = trainline.Passenger(birthdate="12/12/1999")
passenger.add_special_card(trainline.TGVMAX, "YourCardNumber")
passenger_2 = trainline.Passenger(birthdate="12/12/1999")


KeyError                                  Traceback (most recent call last)
<ipython-input-1-7fa648000ffe> in <module>
      3 passenger = trainline.Passenger(birthdate="12/12/1999")
      4 passenger.add_special_card(trainline.TGVMAX, "YourCardNumber")
----> 5 passenger_2 = trainline.Passenger(birthdate="12/12/1999")

~/git_core/trainline-python/trainline/__init__.py in __init__(self, birthdate, cards)
    320             if card not in _AVAILABLE_CARDS:
    321                 raise KeyError("Card '{}' unknown, [{}] available".format(
--> 322                     card, ",".join(_AVAILABLE_CARDS)))
    323         self.cards = cards
    324 

KeyError: "Card '{'reference': 'SNCF.HappyCard', 'number': 'YourCardNumber'}' unknown, [SNCF.CarteEnfantPlus,SNCF.Carte1225,SNCF.CarteEscapades,SNCF.CarteSenior] available"
```

**Comment by tducret** [comment]
Thank you very much @raphael0202 
I learnt a good lesson thanks to you.
I released `v0.0.9` with the fix.

**Comment by raphael0202** [comment]
You're welcome, thanks for the project!
